src/main/resources/application.yml

```diff
@@ -82,6 +82,7 @@ mod-linked-data:
   cache:
     ttl:
       spec-rules: ${CACHE_TTL_SPEC_RULES:18000000}
+      settings-entries: ${CACHE_TTL_SETTINGS_ENTRIES:18000000}
```

Comment:
spring.cache.type=none can be used to disable all caches in karate tests
@pkjacob